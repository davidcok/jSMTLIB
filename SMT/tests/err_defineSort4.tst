; define-sort with not well-sorted expression
(set-logic QF_UF)
(define-sort A () B)
