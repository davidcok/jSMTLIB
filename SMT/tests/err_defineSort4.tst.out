success
(define-sort A () B)
                  ^
(error "No such sort symbol declared: B")
